ArduPlane/quadplane.cpp

```diff
@@ -383,6 +383,15 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     AP_SUBGROUPINFO(qautotune, "AUTOTUNE_",  6, QuadPlane, QAutoTune),
 #endif
 
+    // @Param: TRANS_FAIL_MS
+    // @DisplayName: Quadplane transition failure time
+    // @Description: Maximum time allowed for forward transitions, exceeding this time will cancel the transition and the aircraft will immediately change to QLAND. 0 for no limit.
+    // @Units: ms
```

Comment:
I think we could make this in seconds, as an int16

Reply:
Done.